app/controllers/declaration/DeclarantDetailsPageController.scala

```diff
@@ -33,28 +33,28 @@ import scala.concurrent.{ExecutionContext, Future}
 class DeclarantDetailsPageController @Inject()(
   appConfig: AppConfig,
   override val messagesApi: MessagesApi,
-  authenticate: AuthAction,
+  authenticate: AuthAction, journeyType: JourneyAction,
```

Comment:
Yeah, like Maciej said, please run formatter. This `journeyType` should be in new line

Reply:
Yes... Well spotted. Done! :)